### ISSUE TITLE
Numpy deprecates asscalar

### ISSUE THREAD

**Issue by jranalli** [chmarti1/pyrodoc#3]
Numpy 1.23 [deprecates ](https://numpy.org/doc/1.23/release/1.23.0-notes.html?highlight=asscalar)the function `asscalar`. Installing latest numpy thus breaks our code. 

It's used by PMGI so we need to correct on our end. Notating so we don't forget. 

New usage is given in [old docs](https://numpy.org/doc/1.22/reference/generated/numpy.asscalar.html):
```
np.ndarray.item()
```

**Comment by chmarti1** [comment]
Good catch.  I've gone ahead and searched most of the latest PYroMat code for instances of `asscalar` as well, and I don't see any.  Looks like only the new PMGI code will be affected.
